Framework/API/inc/MantidAPI/GroupingLoader.h

```diff
@@ -0,0 +1,69 @@
+#ifndef MANTID_API_GROUPINGLOADER_H_
+#define MANTID_API_GROUPINGLOADER_H_
+
+#include "MantidAPI/DllConfig.h"
+#include "MantidGeometry/Instrument.h"
+#include <boost/make_shared.hpp>
```

Comment:
I thought they were needed for the `Instrument_const_sptr` but it seems it will compile without them, so they can be removed